package.json

```
@@ -6,7 +6,9 @@
     "install:frontend": "cd frontend && npm install",
     "install:backend": "cd backend && npm install",
     "start:frontend": "cd frontend && npm start",
-    "start:server": "cd backend && npm start"
+    "start:server": "cd backend && npm start",
+    "test:run": "npx cypress run --headed",
+    "test:headless": "npx cypress run"
```

Ao definir scripts no arquivo `package.json`, o `npx` pode ser omitido.

```suggestion
    "test:headless": "cypress run"
```